### ISSUE TITLE
When prompting for a user yes/no answer, use a short [y/n] representation

### ISSUE THREAD

**Issue by christianrondeau** [chocolatey/choco#181]
Here is how yes/no/skip answers are displayed:

```bash
Do you want to run the script?
NOTE: If you choose not to run the script, the installation will
fail.
Skip is an advanced option and most likely will never be wanted.

1) yes
2) no [Default - Press Enter]
3) skip
```

Here is a suggestion of how the prompt could look like:

```bash
Do you want to run the script? (Yes/No/Skip): y
```

In other words: when asking for a yes/no question, use the shorthand, similar to other windows command line tools.

Example:

* xcopy: `Overwrite C:\path\file.txt (Yes/No/All)?`

For more information: https://groups.google.com/forum/#!topic/chocolatey/UVncL7PxXRg

(Moved from https://github.com/chocolatey/chocolatey/issues/689)

**Comment by christianrondeau** [comment]
Hi,

This has been almost fully implemented, but a single detail. See: https://github.com/christianrondeau/choco/blob/22af53bac95cd701cfcc81c7a2a6ef39ceb4cfbb/src/chocolatey/infrastructure/commandline/InteractivePrompt.cs#L58 (Line 58 and 60)

There is a `Log().Info()`, and then a `Console.ReadLine`. That results in two lines. I don't see anywhere in the stack trace a line break, it seems to be within `log4net` that this happens... so in order to achieve what I'm trying to do here, the only way I see I can do it is by:

1. Either I bypass `log4net`, and output the prompt directly to the console, and/or;
2. I then output both the prompt AND the answer in `log4net`; to do this, I'll have to be able to disable console output for that second line (which might be messy), and/or;
3. We skip completely the logging of prompts, and instead we make sure to include in the line *after* the prompt the selection (e.g. `Are you sure? (yes/no): n` then `Selection: no`)

If you know a better way to achieve this, let me know.

**Comment by christianrondeau** [comment]
Well it looks like it's impossible with log4net.

I will then do the direct to console approach, and output the selection after, i.e. (simplified):

Console.Write("prompt: ");
String result = Console.ReadLine();
Log().Info("You have selected " + result");

This means that either 1. We don't see the prompt message in the log, or b. We see it twice in the console.

Unless we create a "prompt" logger, which would behave like the normal logger, without the console appender. Is this worth the additional complexity?

**Comment by ferventcoder** [comment]
maybe...

**Comment by christianrondeau** [comment]
@ferventcoder: Implemented in #204, I personally think it makes sense; most of chocolatey is just output, but this specific line preceding the readline is part of the prompting. The PR will not print in the log file, just the console; then, when we have the input, we print to the log file, but not the console. It looks good on both sides.

**Comment by StefanScherer** [comment]
@ferventcoder I would also love to see a prompt where I can enter `y` or `n` instead of entering numbers. This fits much better with the `-y` command line option.

And then chocolatey is more `apt-get` like.

:+1: for this.

**Comment by ferventcoder** [comment]
@StefanScherer you can enter the value `yes` `no` right now.

**Comment by ferventcoder** [comment]
This allows using the first value. :)

**Comment by StefanScherer** [comment]
@ferventcoder I see. So `1` and `yes` works already, but not `y`:

```
PS C:\Users\vagrant> choco install adobereader
Chocolatey v0.9.9.5
Installing the following packages:
adobereader
By installing you accept licenses for the packages.

adobereader v11.0.10
The package adobereader wants to run 'chocolateyInstall.ps1'.
Note: If you don't run this script, the installation will fail.
Do you want to run the script?
 1) yes
 2) no
 3) print
y
Your choice of 'y' is not a valid selection.
You must select an answer
Do you want to run the script?
 1) yes
 2) no
 3) print
yes
 Switching to German version
 Downloading adobereader 32 bit
...
```

**Comment by ferventcoder** [comment]
Right, the `y` is what this is about

**Comment by christianrondeau** [comment]
By the way @ferventcoder if you need anything else from me, just let me know (I know you're swamped with the moderation load)

**Comment by ferventcoder** [comment]
No worries, will get this guy in hopefully sometime this month.

**Comment by christianrondeau** [comment]
Rebased and merged with #184 (since they are almost the same set of changes)

**Comment by imranhcoder** [comment]
Problem  Do you want to run the script?([Y]es/[A]ll - yes to all/[N]o/[P]rint):      My Problem Sloved Just     y
